uPortal-groups/uPortal-groups-core/src/main/java/org/apereo/portal/groups/ReferenceIndividualGroupService.java

```diff
@@ -175,17 +175,22 @@ public Iterator findParentGroups(IGroupMember gm) throws GroupsException {
         log.debug("Finding containing groups for member {}", gm.getKey());
         Collection groups = new ArrayList(10);
         IEntityGroup group = null;
-        for (Iterator it = getGroupStore().findParentGroups(gm); it.hasNext(); ) {
+        Iterator it = getGroupStore().findParentGroups(gm);
+        while (it.hasNext()) {
```

Comment:
Out of general curiosity, why was `for (item : iterable)` used elsewhere, but `while` used here?

Reply:
It is to signify when the collection is the right class verses when the collection is being cast. If I had more time, I would consider making it consistent across the class.

Reply:
thanks for the clarification, totally get the need to timebox.